ScoutSuite/providers/azure/rules/rulesets/default.json

```diff
@@ -30,6 +30,18 @@
         "enabled": true,
         "level": "warning"
       }
+    ],
+    "sqldatabase-databases-no-transparent-data-encryption.json": [
+      {
+        "enabled": true,
+        "level": "danger"
```

Comment:
I'd lower this to `warning`. While we always recommend encryption on disk, in cloud environment the likeliness of someone accessing the database service directly is low.